README.md

```diff
@@ -0,0 +1,87 @@
+# multiplex-reservation
+
+## Table of contents
+* [Informacje Ogólne](#informacje-ogólne)
+* [Komunikacja z aplikacją](#komunikacja-z-aplikacją)
+* [Warstwa Service](#warstwa-service)
+* [Generowanie Kina](#generowanie-kina)
+* [Odnośnie wymagań](#odnośnie-wymagań)
+
+## Informacje Ogólne
+
+Aplikacja webowa napisana w języku Java z użyciem frameworka Spring umozliwuająca rezerwację miejsc w kinie typu multiplex.
+Aplikacja nie posiada zaimplementowanego frontendu. Komunikowałem się z nią i testowałem za pomocą aplikacji Postman.
+Aplikacja korzysta z JPA oraz bazy danych H2 pod URL *"http://localhost:8080/console"* z adresem URL JDBC *"jdbc:h2:file:./bazaDanych"*.
+
+Baza danych zbudowana jest za pomocą 6 encji: User, Reservation, Screening, Movie, CinemaRoom, Seat.
+Każda z nich posiada swoje prywatne atrybuty oraz jest połączona z odpowiednimi innymi encjami.
+Każda encja korzysta z Repozytorium, które implementuje *PagingAndSortingRepository*.
+
+## Komunikacja z aplikacją
+
+Apliakcja obsługuje REST API i korzysta z dwóch komunikatów *GET* i *POST* w wielu wariantach:
+
+* Wypisanie pokazów filmowych konkretnego dnia w konkretnym przedziale czasowym za pomocą scieżki: 
+*/multiplex/reservation/screening/interval*
+
+Podać należy 3 paramentry w następującej kolejności: dzień, godzina *Od*, godzina *Do*.  
+Przykładowo: localhost:8080/multiplex/reservation/screening/interval?date=2&timeMin=10&timeMax=17  
```

Comment:
Co to znaczy `date=2`?

Reply:
Miało to symbolizować dzień w miesiącu. Jest to toporne rozwiązanie, ale symbolicznie działa. Mam świadomość, że można to było zrobić lepiej, ale priorytetem dla mnie było samo działanie tej aplikacji i komunikacja z bazą danych.